src/github.com/getlantern/flashlight/feed/feed.go

```diff
@@ -297,3 +289,20 @@ func determineLocale(defaultLocale string) string {
 	}
 	return defaultLocale
 }
+
+func getHTTPClient() (*http.Client, error) {
+	var err error
+	httpClientMutex.Lock()
+	if _httpClient == nil {
+		_httpClient, err = proxied.ChainedNonPersistent("")
```

Comment:
This was an opportunistic fix.  I noticed that feed was constructing a new http.Client every time it fetched feeds, meaning that it didn't take advantage of keep-alives.  By making the client a global and lazily initializing it, we can take advantage of persistent connections.

cc: @atavism